Builds/CMake/CMakeFuncs.cmake

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.1.0)
 
 if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_BINARY_DIR}")
-  message(WARNING "In-source builds are strongly discouraged.")
+  message(WARNING "In-root builds are strongly discouraged.")
```

Comment:
FWIW - I like your new term. You should patent it ;)  To my thinking, the hallmark of an "out of source" build is that none of the source files being built are children of the current directory...so I would say your ./build rooted build qualifies. When I want to do an out of source build, I usually just make a new directory in the source tree and it helps if .gitignore knows to ignore it.